include/grpc/impl/channel_arg_names.h

```diff
@@ -51,11 +51,16 @@
  * server will close the connection. Int valued, milliseconds. INT_MAX means
  * unlimited. Defaults to INT_MAX. */
 #define GRPC_ARG_MAX_CONNECTION_IDLE_MS "grpc.max_connection_idle_ms"
-/** Maximum time that a channel may exist. Int valued, milliseconds.
- * INT_MAX means unlimited. Defaults to INT_MAX. */
+/** Maximum time in milliseconds that a channel may exist. Int valued, milliseconds.
```

Comment:
Let's just say - 
"Maximum amount of time in milliseconds that a connection may exist before it will be gracefully shut down. Refer https://github.com/grpc/proposal/blob/master/A9-server-side-conn-mgt.md for more details. Int valued, defaults to INT_MAX (disabled)."